### ISSUE TITLE
implement circular_layout

### ISSUE THREAD

**Issue by maximelucas** [xgi-org/xgi#359]
For the record, here's networkx's:
https://networkx.org/documentation/stable/reference/generated/networkx.drawing.layout.circular_layout.html

**Comment by thomasrobiglio** [comment]
I had done that [here](https://github.com/thomasrobiglio/xgi/blob/new_layouts/xgi/drawing/layout.py) completely relying on networkx, do we want to do it this way or writing the code from scratch? the spring layout functions are based on nx as well

**Comment by maximelucas** [comment]
Oh nice! That's on your fork, not on the main right?
I would probably rewrite the circular layout as it's so simple and avoids the conversion from hypergraph to graph.

**Comment by thomasrobiglio** [comment]
Yes, it's on my fork. Agreed on rewriting the circular layout (I'd say the spiral as well). I'll do that, write the tests and open the PR.

**Comment by maximelucas** [comment]
Great thanks Thomas!!
